tensorflow/tools/ci_build/ci_test_on_install.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
This is because the android option is valid for the ci_build.sh, but is not for the test-on-install.